useEffect/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import './styles.css';
+
+function App() {
+  const [position, setPosition] = useState(0);
+
+  useEffect(() => {
+    setInterval(() => {
+      const moveDistance = 5;
+      setPosition((prevPosition) => prevPosition + moveDistance);
+    }, 1000);
+  }, []);
```

Comment:
1. setInterval을 사용하고 나서는 clearInterval을 통해서 컴포넌트가 리렌더링(update)되거나, 종료되는 경우(unmount)에 실행중인 interval을 제거해주기! 이렇게 해주지 않으면 setInterval이 여러번 실행되서 문제가 생길 수 있음. + useInterval 훅에 대해서 찾아보기!
2. moveDistance와 같은 상수값은 대문자로 작성하기!
3. setInterval의 2번째 인자로 들어가는 delay 값들은 숫자가 커질 경우 간략하게 작성하기!
1분을 나타낸다고 하면 `60_000` 또는 `1000 * 60` 또는 `60000 // 1분` 또는 변수를 미리 선언해두고 `MINUTE` 과 같이 표현하면 좋다!

```suggestion
  useEffect(() => {
    const id = setInterval(() => {
      const MOVE_DISTANCE = 5;
      setPosition((prevPosition) => prevPosition + MOVE_DISTANCE);
    }, 1_000);
    
    return () => {
      clearInterval(id);
     }
  }, []);
```

Reply:
1. useInterval은 사용법을 좀 더 익히고 적용해보겠습니다.